docs/DevelopmentGuide.md

```
@@ -378,6 +378,22 @@ This ensures that the project can still be built and the tests pass with the min
 $ nox -s minimums
 ```
 
+:::{note}
+Both `tests` and `minimums` leave the heavy optional dependencies out by default: the `torch` group is not installed,
+and the tests marked `torch` are deselected.
+That keeps a local run cheap, at the cost of exercising less than CI does.
+Pass `--full` to opt in:
+
+```console
+$ nox -s tests -- --full
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)